Purchases/Purchasing/StoreKitAbstractions/SK1StoreTransaction.swift

```diff
@@ -71,4 +67,19 @@ extension SKPaymentTransaction {
         return identifier
     }
 
+    fileprivate var quantity: Int {
+        return self.maybePayment?.quantity ?? 1
```

Comment:
I'm on the fence about this. I don't _think_ we can assume `1` 😮‍💨
Is it better to do `quantity: Int?` then? 
Looking for input (cc @aboedo)

Reply:
Oh thanks! I meant to ask about that.

Reply:
we don't currently support multi-quantity purchases on iOS, so quantity = 1 should be fine. Otherwise making it optional works, since we don't actually set the quantity on the payment right now, so it defaults to 1.

Reply:
let's add a comment, though, to be very explicit about the fact that we don't currently support multi-quantity purchases and are hard-coding a 1 for now

Reply:
Thanks for the context, totally agree with the comment.

Reply:
Sounds good, defaulting to 1 and leaving a comment for now then.

Reply:
So to clarify:
- Added a comment on the public API in `StoreTransactionType` (which gets inherited by the public `StoreTransaction`)
- Added a comment on `SKPaymentTransaction.quantity`
- The `?? 1` is to deal with the fact that the `payment` might not be there contrary to Apple's API. If that's `nil` (which is supposed to be a bug), then that's the only case we default to 1, which seems like a reasonable default 
- That's the only place where it could be `nil`, which is why I didn't think making it `Int?` was worth it.